src/commands/mod.rs

```diff
@@ -1289,13 +1289,15 @@ fn cmd_show(ui: &mut Ui, command: &CommandHelper, args: &ShowArgs) -> Result<(),
         };
     let template_string = format!(
         r#"
-            "Commit ID: " commit_id "\n"
-            "Change ID: " change_id "\n"
-            "Author: " author " (" {author_timestamp_template} ")\n"
-            "Committer: " committer " (" {committer_timestamp_template} ")\n"
-            "\n"
-            if(description, description, {DESCRIPTION_PLACEHOLDER_TEMPLATE} "\n")
-            "\n""#,
+            concat(
+              "Commit ID: {{commit_id}}\n",
+              "Change ID: {{change_id}}\n",
+              "Author: {{author}} ({{{author_timestamp_template}}})\n",
+              "Committer: {{committer}} ({{{committer_timestamp_template}}})\n",
```

Comment:
Mixing up Rust's `format!` and this interpolation syntax gets confusing.

Reply:
Yeah, this should be resolved by template aliases #1190. (WIP)